modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.scss

```diff
@@ -33,7 +33,7 @@
 		&-save-button {
 			align-items: center;
 			display: flex;
-			height: 32px;
+			height: 40px;
```

Comment:
Hi @Artartur, the correct height for the button is indeed 32px, to follow the same pattern applied in `RightSidebarObjectFieldDetails`. The problem here is that we are missing the class name `lfr-objects__model-builder-right-sidebar-object-relationship-title-delete-button` in the trash button inside `RightSidebarObjectRelationshipDetails`.

I also noticed we have a problem with the class name for the save button inside `RightSidebarObjectDefinitionDetails`. It is missing the word title before "save-button", can you please fix that too?

Thanks!

Reply:
Sure, done! Thank you for the review.